### PR TITLE
feat: update fundamental-styles to 0.11.0-rc.34

### PR DESCRIPTION
**Pull request by meganaconley**
### Description
- Update `Select` a11y and add empty option
- Update `Avatar` background color class name
- Update `Pagination` selected link markup
- Add `DatePicker` 'Today' button
- Update `SideNavigation` markup for a11y
- Update `Table` classnames
- Use popover for `InlineHelp`

### CODE REVIEW

File: src/Calendar/Calendar.js [hunk 1]
```diff
@@ -740,6 +756,8 @@ Calendar.propTypes = {
     }),
     /** Additional props to be spread to the month\'s `<table>` element */
     monthListProps: PropTypes.object,
+    /** Set to **true** if the Today button should be displayed */
+    showToday: PropTypes.object,
```
Comment (prsdthkr) [inline src/Calendar/Calendar.js:760]: Trying to do this using a localizable `todayLabel`: https://github.com/SAP/fundamental-react/pull/964

Comment (meganaconley) [reply src/Calendar/Calendar.js:760]: Ah yeah I forgot to localize the label, thanks

File: src/InlineHelp/InlineHelp.js [hunk 2]
```diff
@@ -55,11 +70,11 @@ InlineHelp.propTypes = {
     'right',
     'left',
     'bottom-center' */
```
Comment (prsdthkr) [inline src/InlineHelp/InlineHelp.js:72]: Placement values should be updated; or point to `Popover > props > placement` documentation

File: src/InlineHelp/__stories__/InlineHelp.stories.js [hunk 3]
```diff
@@ -10,13 +9,6 @@ export default {
...
-export const withLabel = () => (
-    <FormLabel isInlineHelp>
-        Label Text
-        <InlineHelp text='default' />
-    </FormLabel>
-);
-
 export const bottomRight = () => (
     <InlineHelp placement='bottom-right' text='Lorem ipsum dolor sit amet, consectetur adipiscing.' />
 );
```
Comment (prsdthkr) [inline src/InlineHelp/__stories__/InlineHelp.stories.js:14]: Placement values should be updated i.e. `bottom-end` etc.

File: src/TreeView/__stories__/__snapshots__/TreeView.visual.storyshot [hunk 4]
```diff
@@ -533,39 +533,39 @@ exports[`Storyshots Visual Avatar 1`] = `
       className="fddocs-container"
     >
       <span
```
Comment (jbadan) [inline src/TreeView/__stories__/__snapshots__/TreeView.visual.storyshot:535]: If you merge in master and rerun your storybook tests this bug should be resolved https://github.com/SAP/fundamental-react/pull/1092

**Comment by jacobdevera** [comment]
side-nav
It appears the `fd-button` styling is taking precedent over `fd-nested-list__button` to apply a `border-width` to the chevron button here, unlike in Styles:
side-nav-styles

Everything else looks good.

**Comment by prsdthkr** [comment]
fixes #1014 
fixed #920

**Pull request merged**